.eslintrc.json

```diff
@@ -1,6 +1,231 @@
 {
   "extends": "./node_modules/aurelia-tools/.eslintrc.json",
   "rules": {
-    "no-new-func": 0
+    "accessor-pairs": "error",
+    "array-bracket-spacing": [
```

Comment:
are those all needed here? as you see above, we do already extend ./node_modules/aurelia-tools/.eslintrc.json,